fieldservice_change_management/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020 Pavlov Media <https://www.pavlovmedia.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Field Service - Change Management',
+    'summary': 'Manage Change Logs on Locations',
+    'author': 'Patrick Wilson: Pavlov Media',
+    'website': 'https://www.pavlovmedia.com',
+    'category': 'Extra Tools',
+    'version': '12.0.1.0.0',
+    'license': 'Other proprietary',
+    'depends': ['mail',
+                'fieldservice',
```

Comment:
```suggestion
    'depends': [
        'fieldservice',
```

`fieldservice` already depends on `mail`.